nnvm/tests/python/frontend/onnx/test_forward.py

```diff
@@ -712,6 +712,104 @@ def test_constantfill():
     verify_constantfill(False, (2, 3, 4, 5), (2, 3, 4, 5), 10, 'float32')
     verify_constantfill(True, (2, 3, 4, 5), (2, 3, 4, 5, 4, 5, 6), 10, 'float32', extra_shape=(4, 5, 6))
 
+
+def verify_pad(indata, outdata, pads, value=0.0):
+    indata = np.array(indata).astype(np.float32)
+    outdata = np.array(outdata).astype(np.float32)
+    node = helper.make_node(
+        'Pad',
+        inputs=['input'],
+        outputs=['output'],
+        mode='constant',
+        pads=pads,
+        value=value
+    )
+    graph = helper.make_graph([node],
+                              'pad_test',
+                              inputs = [helper.make_tensor_value_info("input",
+                                            TensorProto.FLOAT, list(indata.shape))],
+                              outputs = [helper.make_tensor_value_info("output",
+                                            TensorProto.FLOAT, list(outdata.shape))])
+    model = helper.make_model(graph, producer_name='pad_test')
+
+    for target, ctx in ctx_list():
+        tvm_out = get_tvm_output(model, indata, target, ctx, outdata.shape, 'float32')
+    tvm.testing.assert_allclose(outdata, tvm_out)
+
+def test_pad():
+    verify_pad([[0., 1.], [2., 3.]], [[0., 0., 1.], [0., 2., 3.]], [0, 1, 0, 0], 0.0)
```

Comment:
suggest use random values instead. Also use numpy computed result instead of hand crafted.